package.json

```diff
@@ -38,7 +38,7 @@
     "test:e2e:firefox:snaps": "SELENIUM_BROWSER=firefox node test/e2e/run-all.js --snaps",
     "test:e2e:firefox:nft": "SELENIUM_BROWSER=firefox node test/e2e/run-all.js --nft",
     "test:e2e:single": "node test/e2e/run-e2e-test.js",
-    "test:e2e:report": "jrm ./test/test-results/e2e.xml \"./test/test-results/e2e/*.xml\"",
+    "test:e2e:report": "node ./test/e2e/e2e-process-report.js && jrm ./test/test-results/e2e.xml \"./test/test-results/e2e/*.xml\"",
```

Comment:
Process the xml report before merging, so it can be uploaded to circle ci for test insights